src/components/Dropdowns/HelpDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import { ArrowRightOutlined } from "@ant-design/icons";
+import { Space } from "antd";
+import React, { ReactElement } from "react";
+
+import { VisuallyHidden } from "../../styles/utils";
+
+import AccessibleDropdown from "./AccessibleDropdown";
+import DropdownItem, { DropdownItemList } from "./DropdownItem";
+
+export default function HelpDropdown(): ReactElement {
+  const makeOnButtonClick = (link: string) => {
+    return () => {
+      window.open(link, "_blank", "noopener noreferrer");
+    };
+  };
+
+  const dropdownContent = (
+    <DropdownItemList>
+      <DropdownItem
+        key={"issues"}
+        onClick={makeOnButtonClick("https://github.com/allen-cell-animated/nucmorph-colorizer/issues")}
```

Comment:
why isn't this just wrapping a link element?

Reply:
I made these use an `onClick` because originally the help links didn't respond to keyboard-based click interactions. It was also easier to style them like the other dropdowns by making them into buttons.

Reply:
Mm, correction, I think I want this to be a link element too now. I'd like for the user to be able to see what link they'll be sent to.